app/src/main/java/ch/epfl/skysync/screens/admin/AdminChat.kt

```diff
@@ -0,0 +1,40 @@
+package ch.epfl.skysync.screens.admin
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.navigation.NavHostController
+import ch.epfl.skysync.components.GroupChat
+import ch.epfl.skysync.navigation.AdminBottomBar
+import ch.epfl.skysync.navigation.Route
+import ch.epfl.skysync.viewmodel.ChatViewModel
+
+@Composable
+fun AdminChatScreen(navController: NavHostController, viewModel: ChatViewModel) {

```

Comment:
Is the admin chat any different from the crew/pilot one ? If not, I advise to keep a single chat/text screen directly in the `screens` folder (as it is done for the login screen for example)

Reply:
they are the same. I did two different screens in order to not have to check if the user is an admin or not every time

Reply:
Because the bottom bar is different, I see, then it makes sense to have two separate screens